enterprise/frontend/src/metabase-enterprise/transforms/components/QueryEditor/EditorBody/EditorBody.tsx

```diff
@@ -81,15 +92,23 @@ export function EditorBody({
   const [isResizing, setIsResizing] = useState(false);
   const reportTimezone = useSetting("report-timezone-long");
 
+  const { height: windowHeight } = useWindowSize();
+  const headerHeight = getHeaderHeight(isNative);
+  const editorHeight = Math.min(
```

Comment:
You could move this to the function as well to avoid extra variables in the component. Like `getInitialEditorHeight`.